src/core/sagas/utils.js

```diff
@@ -14,6 +12,11 @@ export function createErrorHandler(id: string): typeof ErrorHandler {
 }
 
 // Convenience function to extract API info.
-export function getApi(state: {| api: ApiStateType |}): ApiStateType {
+export function getApi(state) {
   return state.api;
 }
+
+// Convenience function to extract authentication info.
+export function getAuth(state) {
```

Comment:
I guess so. I was thinking it was cleaner not to just extract the entire state in case we want to manipulate certain things in the future, but I guess we would have state extraction helpers elsewhere that weren't saga-specific, so yeah.